Exercise_A/index.html

```diff
@@ -41,5 +162,6 @@
       If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
       Your custom JavaScript
      -->
-  </body>
+  <!-- </body>-->
+  
 </html>

```

Comment:
For the rest of this website, its done quite well but i would suggest maybe change some of the positioning using the Grid and Flex System to lay them out. You can refer to the answer key!